app/config.py

```diff
@@ -587,6 +593,11 @@ class Production(Config):
     if SQLALCHEMY_BINDS['read-db'] is None:
         logging.critical('Missing SQLALCHEMY_DATABASE_URI_READ')
 
+    if os.getenv('NOTIFY_ENVIRONMENT') == 'performance':
```

Comment:
Note - See lines 601 plus below. 

`performance` and `production` env share the same Config class

Reply:
Do we know why they do (not asking you to change anything, just seems weird)

Reply:
I do not - I'm assuming they want Performance to be config the same as Production